go/controller/utils.go

```diff
@@ -18,16 +18,16 @@ import (
 	"k8s.io/client-go/pkg/api/v1"
 )
 
-// AddResourceList add another v1.ResourceList to first's inner quantity.
-// v1.ResourceList is equal to map[string]Quantity
-func AddResourceList(a *v1.ResourceList, b v1.ResourceList) {
+// AddResourceList add another v1.ResourceList to first's inner
+// quantity.  v1.ResourceList is equal to map[string]Quantity
+func AddResourceList(a v1.ResourceList, b v1.ResourceList) {
```

Comment:
`a` is of type `map`, don't need to pass as pointer: https://github.com/golang/go/wiki/CodeReviewComments#copying

I understand from the C programming language's perspective
`AddResourceList(&reqs, container.Resources.Requests)` is more clear than `AddResourceList(reqs, container.Resources.Requests)` about `reqs` has been modified. I think that's fine in Go, since `req` is a `map`.